corpus/macros.txt

```diff
@@ -1,25 +1,41 @@
-============================================
+================================================================================
```

Comment:
Could you remove from this PR all those esthetic changes and just focus on the change mentioned in the PR?

Reply:
Argh, that wasn't intentional. Something about my setup (maybe a differing tree-sitter version?) changed this. I'll fix.